docs/development.rst

```diff
@@ -275,7 +296,7 @@ Code Review
 -----------
 
 Contributions to Girder are done via pull requests with a core developer
-accepting a PR by saying it "Looks good to me" or LGTM. At this point, the
+approving the PR with GitHub review system. At this point, the
```

Comment:
I remember you suggested that. But I saw the Code Review section has few bullet points specifically on what to review. The CONTRIBUTING.md looks like focuses more on the process to make a contribution. Look like these two has different emphasizes.